tests/js/client/server_invalid_options/test-too-few-file-descriptors.js

```diff
@@ -0,0 +1,32 @@
+/*jshint globalstrict:false, strict:false */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test for startup options
+///
+/// DISCLAIMER
+///
+/// Copyright 2010-2012 triagens GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB Inc, Cologne, Germany
+///
+/// @author Jan Steemann
+/// @author Copyright 2019, ArangoDB Inc, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+return {
+  // require an enormous number of file descriptors, hopefully
+  // more than can be made available
+  'server.descriptors-minimum': '10000000000000',
+};
```

Comment:
This file should probably be part of #13206

Reply:
Ooops. Absolutely. Thanks for the hint!

Reply:
Pushed a fix to remove all the unrelated JS files in this PR. 
Thanks again!